test/controllers/return-requirements.controller.test.js

```diff
@@ -151,12 +152,17 @@ describe('Return requirements controller', () => {
   })
 
   describe('GET /return-requirements/{sessionId}/points', () => {
+    beforeEach(async () => {
+      Sinon.stub(SelectPointsService, 'go').resolves({
+        id: '8702b98f-ae51-475d-8fcc-e049af8b8d38', pageTitle: 'Select the points for the requirements for returns'
+      })
+    })
```

Comment:
Having removed lines above, here we need one!
```suggestion
    })

```